0000-product.md

```diff
@@ -0,0 +1,474 @@
+# Summary
```

Comment:
Missing the following from the top 

Feature Name: Product
Start Date: TBD
RFC PR: TBD